modules/core/src/main/java/org/apache/ignite/cache/CacheMetrics.java

```diff
@@ -243,13 +243,27 @@
      */
     public int getSize();
 
+    /**
+     * Gets number of non-{@code null} values in the cache.
+     *
```

Comment:
Can this method return -1? If yes, what it means?

Reply:
Yeah, it works as `getSize()` method above in such case. See `CacheMetricsImpl.getEntriesStat()` for more details. It will return `-1` when `GridCacheContext` throws exception. I decided to left javadoc similar with `getSize`, which hasn't information about this behaver.